lib/views/help/freedom_of_information_scotland_act.html.erb

```diff
@@ -0,0 +1,160 @@
+  </h4>
+  <p>An authority does not have to provide you with information that is already published somewhere else e.g information that’s available to everyone online or that is in a book or newspaper. This includes information referred to in a public authority’s publication scheme. The publication scheme sets out a public authority’s commitment to make some types of information routinely available on request. Information can be considered to be accessible by other means even if there is a charge to access it.
+  </p>
+  <h4 id="prohibited">
+    Prohibitions on disclosure (Section 26)
+    <a href="#prohibited">#</a>
+  </h4>
+  <p>Information can be withheld if disclosure is prohibited by law.</p>
+  <h4 id="court_records">
+    Court records (Section 37)
+    <a href="#court_records">#</a>
+  </h4>
+  <p>Court records are exempt from disclosure, including records of what was said in court, or documents that have been submitted by parties involved in a court case. There are separate access regimes for getting access to these.</p>
+ <h4 id="personal_information">
+  Personal Information (Section 38)
+  <a href="#personal_information">#</a>
+ </h4>
+ <p>An authority can refuse to provide you with your own or someone else's personal information. The exemption only applies if the person can be identified e.g. because the data includes the person’s name or an identifier such as National Insurance number or email address. Although this is an absolute exemption, the authority might carry out a public interest test when considering whether it would be fair to release third party personal information.</p>
+  <h2 id="unhappy">
+    What if I am unhappy with my response?
+    <a href="#unhappy">#</a>
+  </h2>
+  <p>If you are unhappy with a response that you have received, or you have not received a response at all, you can ask the authority to conduct an internal review. You need to ask for a review no later than 40 working days after the date that you received a response, or 60 working days after you made your request, in the event that the authority didn't reply at all.</p>
+  <p>
+  If you are unhappy with the outcome of the internal review, or if the authority has not responded to your request for a review, you can appeal to the Scottish Information Commissioner. You need to submit your appeal within 6 months of the date that you received the outcome of your request for a review. The Scottish Information Commissioner can not consider an appeal about a response to a request that was made to the Scottish Information Commissioner.
+  </p>
+  <h2 id="Appeal">
+    How do I appeal to the Scottish Information Commissioner?
+    <a href="#appeal">#</a>
+  </h2>
+  <p>
+  You can contact the Scottish Information Commissioner by email on enquiries@itspublicknowledge.info or by post:
+  </p>
+  <p>
+  Scottish Information Commissioner<br>
+  Kinburn Castle<br>
+  Doubledykes Road<br>
+  St Andrews<br>
+  Fife<br>
+  KY16 9DS<br>
+  </p>
```

Comment:
```suggestion
  <p>
  You can contact the Scottish Information Commissioner by email at: <b><a href="mailto:enquiries@itspublicknowledge.info">enquiries@itspublicknowledge.info</a></b>, or by post:
  </p>
  <p>
  Scottish Information Commissioner<br>
  Kinburn Castle<br>
  Doubledykes Road<br>
  ST ANDREWS<br>
  KY16 9DS<br>
  </p>
  <p>
  You can also contact them for advice, by calling their helpline on <b>01334 464 610</b>, or if you use <abbr title="British Sign Language">BSL</abbr>, by using <a href="http://contactscotland-bsl.org">Contact Scotland BSL</a>.
  </p>
```

I thought it'd be helpful to call out the helpline here [^1].

Pedantry: Royal Mail [confirms](https://www.royalmail.com/find-a-postcode) the post town is St Andrews, so we can reduce some of the address by stopping there. 🙂 [^2]

[^1]: The BSL link here is the Scottish Government funded VRS, which OSIC are required to promote (as per their [BSL plan](https://www.itspublicknowledge.info/strategic-and-operational-plans). It's important that we call this out.
[^2]: Technically, they say "ST. ANDREWS", but the full stop is probably superfluous.